### ISSUE TITLE
Can you make a how to tutorial for posts view count

### ISSUE THREAD

**Issue by lakshmanbasnet** [sharu725/blog-v3#4]
* you have used google analytics to show the number of views for posts, can you please make one blog post on setting up this all
* I check our blog regularly and I follow your tips in my jekyll site

**Comment by sharu725** [comment]
@lakshmanbasnet thanks for asking. I have thought about it, many times actually. It is a little difficult for an average user. But, I did it using a tutorial (after many trials). It didn't work for me initially because the tutorial was a little old and Google changed a lot of things in a very short span. They might make changes and make it easier in the future.  
Thanks for the suggestion,  I will try to cover it in my future posts.

**Comment by lakshmanbasnet** [comment]
yeah, I also got one tutorial but I tried and tried, but couldn't succeed. So, I posted here, seeking help from you. It would be really useful, if you could make one. Waiting for it eagerly. and thanks for instant reply!

**Comment by sharu725** [comment]
Been there. Someday I will.
